src/index.js

```diff
@@ -0,0 +1,97 @@
+import {
+  addTask, deleteTask, editTask, clearCompleted,
+} from './status.js';
```

Comment:
Please import all exports from a module in one variable as you might not know what is exported from the module so instead of using import { addTask, deleteTask, editTask, clearCompleted, } from './status.js'; use 
import  taskStatus  from './status.js'; 
and to use the functions inside it use taskStatus.addTask.

Reply:
Thank you for the review. However, I didn't understand so well what you actually meant